dev/create-release/spark-rm/Dockerfile

```diff
@@ -61,9 +61,9 @@ RUN apt-get clean && apt-get update && $APT_INSTALL gnupg ca-certificates && \
   apt-get update && \
```

Comment:
Since you're here, can you also fix the comments at `dev/create-release/do-release-docker.sh`:

```
  -j [path]   : path to local JDK installation to use for building. By default the script will
                use openjdk8 installed in the docker image.
```

Reply:
Okay, let me fix it.

Reply:
Done